### PR TITLE
Compute file hashes on request via `it` and compare if has changed ##bin

### PR DESCRIPTION
**Pull request by radare**

### CODE REVIEW

File: libr/core/cmd_info.c [hunk 1]
```diff
@@ -457,20 +457,48 @@ static int cmd_info(void *data, const char *input) {
...
+						char *s = r_str_prefix_all (old, "- ");
+						r_cons_printf ("%s\n", s);
+						free (s);
+						s = r_str_prefix_all (info->hashes, "+ ");
+						r_cons_printf ("%s\n", s);
+						free (s);
+						break;
+					}
+				}
+				if (input[1] == 'j') {
```
Comment (ITAYC0HEN) [inline libr/core/cmd_info.c]: // "itj"

Comment (radare) [reply libr/core/cmd_info.c]: Just press the pencil button

**Comment by ITAYC0HEN** [comment]
what "it" stands for?

**Comment by radare** [comment]
it was the only letter that was not used for anything, is a bad command, but the pr was stalled there for a month and nobody was reviewing or suggesting any better option. but just complaining because they were unable to get the hash of the file. so at least now it works :P

**Comment by ITAYC0HEN** [comment]
problematic indeed. but that's what we have haha

**Pull request merged**